cgsmiles/resolve.py

```diff
@@ -169,17 +169,18 @@ def edges_from_bonding_descrpt(self):
             for _ in range(0, self.meta_graph.edges[(prev_node, node)]["order"]):
                 prev_graph = self.meta_graph.nodes[prev_node]['graph']
                 node_graph = self.meta_graph.nodes[node]['graph']
-                edge, bonding = generate_edge(prev_graph,
-                                              node_graph)
-
+                try:
+                    edge, bonding = generate_edge(prev_graph,
+                                                  node_graph)
+                except LookupError:
+                    continue
```

Comment:
If I get it correctly, this deals with leftover bonding descriptors?

Reply:
No this one deals with edges at the higher levels that do not generate an edge at the lower levels. Essentially, it goes back to the problem that Martini bonded connectivity does not necessarily follow all-atom connectivity